microcosm_pubsub/producer.py

```diff
@@ -230,13 +230,8 @@ def configure_sns_producer(graph):
     except NotBoundError:
         opaque = None
 
-    if graph.config.sns_producer.skip is None:
```

Comment:
Note that with this change, running `runserver` or the equivalent and making HTTP requests with pubsub side effects *WILL* error out if there's no where to publish to. This means that running with `LocalStack` is more or less mandatory.

I'm open to alternatives.

Reply:
It would be nice not to have to worry about this when debugging locally. can we keep the default and change the ansible default instead?

Reply:
We can change the globe config defaults probably.